setup_env.sh

```diff
@@ -4,11 +4,8 @@ export ZETH=`pwd`
 export ZETH_CONTRACTS_DIR=$ZETH/zeth_contracts/contracts
 export ZETH_DEBUG_DIR=$ZETH/debug
 
-mkdir -p $ZETH/notestore
-export ZETH_NOTESTORE=$ZETH/notestore
-
-mkdir -p $ZETH/trusted_setup
-export ZETH_TRUSTED_SETUP_DIR=$ZETH/trusted_setup
+mkdir -p $ZETH/zeth_setup
+export ZETH_SETUP_DIR=$ZETH/zeth_setup
```

Comment:
Not strictly part of this change, and just a small point, but the following would avoid duplicating the value of `ZETH_SETUP_DIR`
```
export ZETH_SETUP_DIR=$ZETH/zeth_setup
mkdir -p $ZETH_SETUP_DIR
```

Reply:
Indeed, that's a good point, thanks.